src/rpc/api.ts

```diff
@@ -209,7 +209,7 @@ export namespace Api {
     type: EventType;
     ledger: number;
     ledgerClosedAt: string;
-    pagingToken: string;
+    cursor: string;
```

Comment:
It appears that the js-stellar-sdk is being updated to only support the rpc v22 api.

What's the plan for how devs will coordinate the use of tools like the js-stellar-sdk and rpc that may upgrade at different versions? Or coordinate the use of dapps or other products that interact with both testnet and mainnet?

The update to only v22 means there's no version of js-stellar-sdk that can be used with both, so a js-stellar-sdk and rpc pair that interact will have to be stopped, updated, and started.

Reply:
> coordinate the use of tools like the js-stellar-sdk and rpc that may upgrade at different versions? 

When an RPC upgrades, SDK clients must upgrade accordingly. This has always been the case and is pretty standard practice?

> interact with both testnet and mainnet

A common practice in JS is [package aliasing](https://stackoverflow.com/a/56495651) to have multiple versions of a dependency running at the same time. They would use the v22-compatible version for testnet for breaking changes they're encountering and switch off.

> there's no version of js-stellar-sdk that can be used with both

Again, this has always been the case. If we want to change that, that's one thing, but this is the same exact upgrade pattern that the network has always experienced with regards to breaking API changes to backend services. We could feasibly bundle two versions under separate namespaces, but I'm still not sure how that would solve the problem better than having two versions installed yourself.